src/NUnitTestAdapter/TestLogger.cs

```diff
@@ -48,7 +48,7 @@ public void LoadingAssemblyFailedWarning(string dependentAssembly, string source
 
         public void NUnitLoadError(string sourceAssembly)
         {
-            SendErrorMessage("NUnit failed to load " + sourceAssembly);
+            SendInformationalMessage("NUnit failed to load " + sourceAssembly);
```

Comment:
The other place where this is called is in the TestExecutor in the else clause for `if (loadResult.GetAttribute("runstate") == "Runnable")`, so it does look like it should be an informational message. Should the method be renamed appropriately or at least a comment placed on the method?